docs/mail.rst

```diff
@@ -0,0 +1,88 @@
+Email basics
+============
+
+The big picture
+---------------
+Sending email involves lots of components.
+
+.. image:: https://res.cloudinary.com/shaka/image/upload/v1542098296/ansible_mailserver/simple_mail_1.png
+
+Components
+----------
+
+Mail Transfer Agent
+^^^^^^^^^^^^^^^^^^^
+Within Internet message handling services (MHS), a message transfer agent or
+mail transfer agent (MTA) or mail relay is software that transfers electronic
+mail messages from one computer to another using a client–server application
+architecture. An MTA implements both the client (sending) and server (receiving)
```

Comment:
```suggestion
architecture. A MTA implements both the client (sending) and server (receiving)
```

Reply:
I was not clear enough, you reviewed wikipedia :laughing:, it's all my fault.